requirements.in

```diff
@@ -1,4 +1,4 @@
-Django<3.3
+Django<6
```

Comment:
Is all the grantnav code compatible with Django 5 too?

Reply:
Have now set to <5 in https://github.com/ThreeSixtyGiving/grantnav/pull/1156/commits/0c40ad3ec1a5a44a00a5740c4af4fbddeb52f50c